src/utils/i18n.js

```diff
@@ -0,0 +1,7 @@
+import translations from '../assets/locales/i18n';
+
+const DEFAULT_LANG = 'en';
+
+export default function getTranslations(lang) {
+    return translations[lang.substr(0, 2).toLowerCase() || DEFAULT_LANG];
```

Comment:
2 chars only? what about `zh-hans` and `zh-hant`?

Reply:
this is why it's called 'base setup' :trollface: 

(I'll think about it 🎉)